examples/nodejs/aws/eventbridge/run-cli.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
can we rename this file to `run-bash-cli-app.sh`, and the other one to `run-typescript-cli-app.sh`?